src/microbe_stage/organelle_components/BindingAgentComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿public class BindingAgentComponent : IOrganelleComponent
```

Comment:
Should have a comment that this does nothing else except is useful to check if a microbe should be able to bind.